editoast/src/models/round_trips.rs

```diff
@@ -0,0 +1,87 @@
+use crate::models::prelude::*;
+use crate::views::pagination::PaginationStats;
+use database::DbConnection;
+use diesel::ExpressionMethods;
+use diesel::QueryDsl;
+use diesel_async::RunQueryDsl;
+use editoast_derive::Model;
+use futures_util::stream::TryStreamExt;
+use std::ops::DerefMut;
+
+#[derive(Clone, Debug, Model)]
+#[model(table = database::tables::train_schedule_round_trips)]
+#[model(gen(batch_ops = c, list))]
+pub struct TrainScheduleRoundTrips {
+    pub id: i64,
+    /// First ID of the train schedule of this round trip
+    pub left_id: i64,
+    /// Second ID of the train schedule of this round trip
+    /// This is `None` for one-way trains
+    pub right_id: Option<i64>,
+}
+
+impl DeleteBatch<i64> for TrainScheduleRoundTrips {
```

Comment:
That's very confusing. In. all our other models `Delete<i64>`, `Retrieve<i64>` refers to the ID. I'm not confident about this exception. How about a new type?

```rust
struct LeftTrain(i64);
impl DeleteBatch<LeftTrain> for ...
```